boards/mro/ctrl-zero-f7/src/board_config.h

```diff
@@ -175,7 +175,7 @@
 #define BOARD_ADC_USB_CONNECTED	       (px4_arch_gpioread(GPIO_OTGFS_VBUS))
 #define BOARD_ADC_USB_VALID            BOARD_ADC_USB_CONNECTED
 #define BOARD_ADC_SERVO_VALID          (1)	/* never powers off the Servo rail */
-#define BOARD_ADC_BRICK_VALID          (!px4_arch_gpioread(GPIO_nVDD_BRICK1_VALID))
+#define BOARD_ADC_BRICK_VALID          (px4_arch_gpioread(GPIO_nVDD_BRICK1_VALID))
```

Comment:
So `GPIO_nVDD_BRICK1_VALID` is actually `GPIO_VDD_BRICK1_VALID`?

Reply:
Ok, it is renamed to GPIO_VDD_BRICK1_VALID (if I understood correctly that is what you were after but that would be different than the naming convention of everything else I could see).